builtins/amp-img.md

```diff
@@ -123,6 +123,8 @@ An explicit size of the image, which is used by the AMP runtime to determine the
 
 This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.
 
+Note that the usage of the [`media`](https://www.ampproject.org/docs/reference/common_attributes#media) common attribute is not recommended with `<amp-img>`, as its effects can be created with more refined control by using the `srcset`, `sizes`, and `heights` attributes instead.
```

Comment:
Use of `media` is required for Art Direction (`srcset` is supposed to be same image at diff resolutions and should not be different cropped versions). https://www.ampproject.org/docs/design/responsive/responsive_design#changing-the-art-direction-of-an-image 

Instead of adding this paragraph, my suggestion is linking to https://www.ampproject.org/docs/design/responsive/responsive_design from here.

/cc @bpaduch

Reply:
Agreed.  Add information to the Tips and Tricks section .  Add information about using media for art direction, then link to guide.